bridge_adaptivity/bridge_lti/tests/test_exeptions.py

```diff
@@ -11,25 +10,44 @@
 
 
 from bridge_lti.provider import learner_flow
+from module.tests.test_views import BridgeTestCase
 
 log = logging.getLogger(__name__)
 
 
-class RaisedExceptionUsesCustomTemplateTest(TestCase):
+class RaisedExceptionUsesCustomTemplateTest(BridgeTestCase):
     def setUp(self):
+        super(RaisedExceptionUsesCustomTemplateTest, self).setUp()
         self.rf = RequestFactory()
-        self.url = reverse('lti:launch', kwargs={'collection_id': 1, 'group_slug': 'asd'})
+        self.correct_kw = {'collection_id': self.collection1.id, 'group_slug': self.test_cg.slug}
+        self.not_correct_kw = {'collection_id': self.collection2.id, 'group_slug': self.test_cg.slug}
+        self.url = reverse('lti:launch', kwargs=self.correct_kw)
+        self.not_correct_url = reverse('lti:launch', kwargs=self.not_correct_kw)
 
     @override_settings(DEBUG=False)
-    def test_learner_flow_with_incorrect_collection_id_test(self):
+    def test_learner_flow_with_incorrect_collection_id(self):
         """Check if learner_flow function is called with incorrect collection_id raise proper exception."""
         request = self.rf.post(self.url)
         with pytest.raises(Http404):
             learner_flow(
                 request,
                 lti_consumer=None,
                 tool_provider=None,
-                collection_id=1000
+                collection_id=1000,
+                group_slug=self.test_cg.slug
```

Comment:
trailing with a comma in last tuple/dict line will help to avoid deleting this line in the PR diff.